Source/Core/DolphinWX/Config/GeneralConfigPane.cpp

```diff
@@ -39,10 +39,9 @@ GeneralConfigPane::GeneralConfigPane(wxWindow* parent, wxWindowID id)
 
 void GeneralConfigPane::InitializeGUI()
 {
-	m_frame_limit_array_string.Add(_("Off"));
-	m_frame_limit_array_string.Add(_("Auto"));
-	for (int i = 5; i <= 120; i += 5) // from 5 to 120
-		m_frame_limit_array_string.Add(wxString::Format("%i", i));
+	m_throttler_array_string.Add(_("Disabled"));
```

Comment:
`Infinite` or `Unlimited` or `∞`, maybe? I'm not sure what the best option here is, but you can't just have "Speed: Disabled" in a UI and expect people to take it seriously.